tslearn/utils.py

```diff
@@ -770,471 +770,483 @@ def transpose_or_expand(ts):
     return to_time_series_dataset([transpose_or_expand(Xi) for Xi in X])
 
 
-try:  # Ugly hack, not sure how to to it better
-    import pandas as pd
+def to_sktime_dataset(X):
+    """Transform a tslearn-compatible dataset into a sktime dataset.
 
-    def to_sktime_dataset(X):
```

Comment:
Having the `try: ... except ImportError:` inside the functions is a more standard way of doing it. Done that and removed 1 indentation from all of these functions.

Reply:
OK, but now it seems that on Travis doctests fail if the dependencies are not properly installed.

Reply:
The question is: why do these pass on Azure? Are doctests executed at all?

Reply:
I saw, working on a fix. No I need to add the corresponding pytest option.